README.md

```diff
@@ -1 +1,53 @@
-# This repository is for the Crab Robotics Team robot, CRABOT.
+# READ ME!
+
+## Introduction 
+Welcome to the CRABOT repository! This is where we post edits and new code onto our robot. 
+
+## Features
+We currently have these libraies in our repository.
+
+CrabRobotLibrary:
+ - tell_me_about_your_skills()
```

Comment:
```suggestion
 - `tell_me_about_your_skills()`
```

Maybe make this code formatted?

Reply:
I updated every function name per your suggestion. 👍 ✔️